libraries/Microsoft.Bot.Builder.Dialogs/Memory/PathResolvers/AliasPathResolver.cs

```diff
@@ -46,6 +57,11 @@ public virtual string TransformPath(string path)
             return path;
         }
 
+        /// <summary>
+        /// Verifies if a character is valid for a path.
+        /// </summary>
+        /// <param name="ch">Character to verify.</param>
+        /// <returns><c>true</c> if the character is valid for a path or <c>false</c> if not.</returns>
```

Comment:
```suggestion
        /// <returns><c>true</c> if the character is valid for a path otherwise <c>false</c>.</returns>
```

Similar to a comment at line 5 of the file DialogStateManager.cs

```csharp
        /// <returns><c>true</c> if the item was successfully removed from the dialog state manager;
        /// otherwise, <c>false</c>.</returns>
```

Reply:
good catch